### PR TITLE
[new] Add formula.js w/ npm autoupdate by single package.json

### PR DESCRIPTION
**Pull request by nicolashefti**
Thanks in advance for the peer-review!

FYI: @jalateras, @hmalphettes

# Profile of the lib
 * Git repository (required): https://github.com/sutoiku/formula.js/
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/formulajs
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1.384
   - Count of watchers: 98
   - Count of forks: 122

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@nicolashefti Thanks for your contribution. @pvnr0082t would you like to take a first review?

File: ajax/libs/formulas.js/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "author": "Ismael Ghalimi <ismael@stoic.com>",
+    "name": "formula.js",
+    "version": "1.0.8",
+    "description": "JavaScript implementation of most Microsoft Excel formula functions",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/sutoiku/formula.js.git"
+    },
+    "autoupdate": {
```
Comment (pvnr0082t) [inline ajax/libs/formulas.js/package.json]: @nicolashefti We prefer to use npm auto-update config if the library had been published on npm. Thanks. ref: [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md)

**Comment by pvnr0082t** [comment]
We still need such field as follow:
- filename
- license
- keywords
Please add them. Thanks.

**Comment by pvnr0082t** [comment]
Please use 2-spaces as indent. You can use `tools/fixFormat.js` to fix it. Thanks.

**Comment by pvnr0082t** [comment]
@nicolashefti You can commit `v1.0.8` files under dist folder and put them into `v1.0.8` folder, or remove the version info in package.json file.

**Comment by nicolashefti** [comment]
@pvnr0082t @PeterDaveHello Thanks for being so reactive. Really appreciated! Great work you are doing here.

I updated the PR:
- renaming formula.js to formulajs
- adding missing fields
- switching from git to npm auto-update config
- deleting "version" from package.json

**Comment by PeterDaveHello** [comment]
@nicolashefti Thanks!

File: ajax/libs/formulajs/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "type": "git",
+    "url": "https://github.com/sutoiku/formula.js.git"
+  },
+  "npmName": "formulajs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "formula.js",
+        "formula.min.js"
```
Comment (PeterDaveHello) [inline ajax/libs/formulajs/package.json]: Could we just use a single `"**/*"` here to include all the files?

Comment (nicolashefti) [inline ajax/libs/formulajs/package.json]: @PeterDaveHello PR updated.

**Comment by PeterDaveHello** [comment]
@nicolashefti thank you!

**Pull request merged**